packages/core/src/amazonqFeatureDev/client/featureDev.ts

```diff
@@ -64,6 +67,10 @@ export class FeatureDevClient {
         return await createFeatureDevProxyClient()
     }
 
+    public getOptOutPreference() {
```

Comment:
A quick search shows that this exact code is present in `core/src/codewhisperer/util/commonUtil.ts`. If this is shared code can we extract this.

Reply:
I know as well, this one I didn't see the issue to leave it here private, also I could have just left it as a one liner in the function that makes the request to `sendTelemetryEvent`.

Anyway, for both, sure I can make the changes.